src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/DetailsVA.jsx

```diff
@@ -30,22 +31,25 @@ export default function DetailsVA({ appointment, facilityData }) {
   const isCovid = appointment.vaos.isCOVIDVaccine;
   const header = formatHeader(appointment);
   const isPhone = appointment.vaos.isPhoneAppointment;
+  const serviceType = appointment.vaos.apiData.serviceType;
+  const typeOfCare = getTypeOfCare(serviceType)?.name;
 
   return (
     <>
       <Breadcrumbs>
         <Link to={`/va/${appointment.id}`}>Appointment detail</Link>
       </Breadcrumbs>
-
       <h1>
         <AppointmentDateTime appointment={appointment} />
       </h1>
-
       <StatusAlert appointment={appointment} facility={facility} />
-
+      {typeOfCare && (
+        <div className="vads-u-padding-top--2p5">
+          <b>Type of care:</b> {typeOfCare}
+        </div>
+      )}
```

Comment:
Type of care should be tagged as an h2 so screen readers can scan through the headings enabling users to determine if they want it read out loud or not. It's part of 508.
```suggestion
      {typeOfCare && (
        <>
          <h2 className="vads-u-font-size--base vads-u-font-family--sans vads-u-margin-bottom--0 vaos-word-inline">
            Type of care:
          </h2>
          <div className="vaos-word-inline"> {typeOfCare}</div>
        </>
      )}

      )}
```
In order the have the value stay in the same line as the h2, create a new css class called `vaos-word-inline` in `src/applications/vaos/sass/vaos.scss` with the following 
`.vaos-word-inline {
  display: inline;
}
`